.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3.0.7
+      with:
+        repo-token: ${{ secrets.BEDEVERE_GH_TOKEN }}
+        stale-pr-message: 'This PR is stale because it has been open for 30 days with no activity. Remove stale label or comment or this will be closed in 5 days'
+        stale-pr-label: 'stale-pr'
+        days-before-stale: 30
+        days-before-close: 5
```

Comment:
Are we prepared to close out stale PRs? I think this would require a much wider discussion as that's a change in development approach.

Reply:
There's a prior discussion in https://github.com/python/core-workflow/issues/93 but we have never reached decision. I can ask in Discourse.

Reply:
Asked in https://discuss.python.org/t/decision-needed-should-we-close-stale-prs-and-how-many-lapsed-days-are-prs-considered-stale/4637 but I've made the rest of the suggested change.